tests/src/test/java/alluxio/client/cli/JournalToolTest.java

```diff
@@ -143,7 +143,8 @@ public void dumpHeapCheckpointFromUfsJournal() throws Throwable {
 
   @Test
   @LocalAlluxioClusterResource.Config(confParams = {PropertyKey.Name.MASTER_JOURNAL_TYPE,
-      "EMBEDDED", PropertyKey.Name.MASTER_METASTORE, "HEAP"})
+      "EMBEDDED", PropertyKey.Name.MASTER_INODE_METASTORE, "HEAP",
+      PropertyKey.Name.MASTER_BLOCK_METASTORE, "HEAP"})
```

Comment:
Please add tests for BLOCK_METASTORE=ROCKS and INODE_METASTORE=HEAP and vice versa. Let's make sure all combinations work.

Reply:
Ok, thanks for reviewing this!

Reply:
This test seems just test the Journal tool and only inode metastore is considered, this may be enough and no need to add new test? FYI.